site/docs/theming/best-practices.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Best practices for theming
+layout: DetailTechnical
+---
+
+Salt offers a comprehensive suite of [components](/salt/components) and [patterns](/salt/patterns) designed to streamline both the design and development process. Whilst these resources cover a wide range of use cases, there will inevitably be times when you need to create custom components to meet specific product requirements.
+
+This guide will explore best practices for theming within the Salt design language, highlighting key considerations and best practices.
+
+By adhering to these guidelines
+
+- you will make consuming future Salt updates easier
+- make your components more reusable and composable with other Salt components
+- make your design easier to maintain and understood by others.
+- provide your end user with a cohesive user experience.
+
+## Theming options
+
+Design tokens in the Salt Design language take the form of CSS/Figma variables that enable you to customize the presentation of Salt components.
+
+At its core, theming involves changing preset values that have been carefully chosen to meet design standards and accessibility requirements. Altering one value may necessitate changes to other related values. Therefore, making an uninformed change can lead to unexpected side effects that may not always be immediately visible.
+
+To help you manage the cost of change and choose the model that best suits your product and resources, we offer various entry points with increasing levels of flexibility. The trade-off is that increased flexibility can increase your responsibilities and the depth of knowledge required to maintain it.
+
+### Option 1: Adopt the standard, pre-built, theme
+
+Get your core styles for free.
+
+#### Dos
+
+- Do use the Salt theme "out of the box" and avoid owning any custom styles.
+- Do recognize that adopting Salt aligns your current UX with a maintained standard that conforms to both design and accessibility guidelines. Whilst there is an opportunity cost, deviating from Salt can increase your costs and result in a disparate UI, ultimately affecting the end user's experience.
+
+### Option 2: Compose your theme from pre-made design options
+
+Salt provides pre-made, system-wide [styling options](/salt/theming/index#styling-options) that users can opt into through the SaltProvider. Salt will continue to expand this offering, but it currently offers the easiest, cheapest, and most stable way to customize your UI.
+
+### Option 3: Extend the Salt theme
+
+Each Salt theme consists of 3 layers ([Foundation](/salt/about/glossary#foundation), [Palette](/salt/about/glossary#palette-layer) and [Characteristics](/salt/about/glossary#characteristics-layer) layers) that work together to offer a consistent theming solution across all components:
+
+Whilst this creates the highest level of flexibility it comes with an increased level of responsibility to maintain. If this is what you need , then here are our recommendations:
+
+#### Don'ts
```

Comment:
Dos and Don't tells a few different opinions from Salt, needs very careful team-wide review